### ISSUE TITLE
[BLUEJ-452] Disabling Auto-indent glitches the closing bracket '}'

### ISSUE THREAD

**Issue by neilccbrown** [k-pet-group/BlueJ-Greenfoot#452]
When Auto-indent is disabled, entering a closing bracket '}' will delete all tabs that have been entered on that line previous to it if that closing bracket would otherwise have been auto-indented.

example:

{noformat}
#!java
public void somefunction()
{
	if(true)
	{
		//content here
} //this bracket will eat all previous tabs
}
{noformat}

It may also be helpful to note that I have soft tabs disabled and match brackets also disabled.  This does not happen when tabs are set to insert a truckload of spaces instead of a tab.

On some occasions, it will not eat all previous tabs and instead auto indent anyway (even though it is disabled), replacing my tabs with a bunch of spaces.

Basically, disabling auto indent doesn't work.

---

**Issue metadata**

- Issue type: Bug
- Priority: Low
- Fix versions: 3.0.9
